templ_attr.cif

```diff
@@ -103,13 +103,13 @@ save_rho_coeff
 
 save_rho_kappa
 
-    _definition.update           2021-03-01
+    _definition.update           2024-03-28
     _description.text
 ;
       A radial function expansion-contraction coefficient
-      (κ     = atom_rho_multipole_kappa.base and
-       κ'(l) = atom_rho_multipole_kappa.prime[l])
-      for the atom specified in atom_rho_multipole.atom_label.
+      (κ     = _atom_rho_multipole_kappa.base and
+       κ'(l) = _atom_rho_multipole_kappa.prime[l])
```

Comment:
The `_atom_rho_multipole_kappa.prime*` data names in the electron density dictionary [a] do not contain brackets (i.e. `_atom_rho_multipole_kappa.prime0`   and not `_atom_rho_multipole_kappa.prime[0]`). Could this be a bit misleading, given that other dictionaries do contain data name aliases with brackets (e.g. `_diffrn_orient_matrix.UB[1][1]`)? However, I am not sure that `_atom_rho_multipole_kappa.primel` would work much better in the description since the `l` no longer looks like a variable.

Happy to merge as is if it does not seem like something worth addressing now (or at all).

[a] https://github.com/COMCIFS/Electron_Density_Dictionary

Reply:
Good point. Please accept it for now, as the lack of a leading underscore was causing problems in typesetting the electron density dictionary. The usage of brackets is current at other locations in the electron density dictionary, but I'll bear it in mind as I work through that. There may be cases in other dictionaries where the annotation conventions in the description texts might be misleading. I've started to look at Cif_Rho (cif_rho.dic?) and made an initial PR https://github.com/COMCIFS/Electron_Density_Dictionary/pull/4 to change the _dictionary.namespace.

Reply:
Acknowledged.